tests/test-edit-flow-custom-status.php

```diff
@@ -5,10 +5,13 @@ class WP_Test_Edit_Flow_Custom_Status extends WP_UnitTestCase {
 	protected static $admin_user_id;
 
 	public static function wpSetUpBeforeClass( $factory ) {
+		$EF_Custom_Status = new EF_Custom_Status();
```

Comment:
Thought we ran `init` on all our modules as a part of boostrapping the unit tests. But without this init the tests didn't work as expected (it appeared the custom statuses weren't registered).